jobs/build-kola-containers.Jenkinsfile

```diff
@@ -0,0 +1,224 @@
+def gitref, commit, shortcommit
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+}
+
+properties([
+    pipelineTriggers([
+        [$class: 'GenericTrigger',
+         genericVariables: [
+          [
+           key: 'COREOS_ASSEMBLER_GIT_REF',
+           value: '$.ref',
+           expressionType: 'JSONPath',
+           regexpFilter: 'refs/heads/', //Optional, defaults to empty string
+           defaultValue: ''  //Optional, defaults to empty string
+          ]
+         ],
+         causeString: 'Triggered on $ref',
+         token: 'build-kola-test-containers',
+         tokenCredentialId: '',
+         printContributedVariables: true,
+         printPostContent: true,
+         silentResponse: false,
+         regexpFilterText: '$COREOS_ASSEMBLER_GIT_REF',
+         regexpFilterExpression: 'main|rhcos-.*'
```

Comment:
This will trigger for older RHCOS branches too which don't have the iSCSI tests and don't need to have them built.

Honestly, kinda thinking for now we just build for `main` and even when we branch for e.g. 4.16, those tests can keep using the `:main` tag of the container image. If we feel we need to branch the images too, we can do that then. But it's worth mentioning it in their READMEs in cosa.

Reply:
these image will probably have very little change, other than the occasionnal base image bump, so I agree sticking to one main image is fine, until we find out we need to change that